java/src/test/java/com/saucelabs/simplesauce/BaseConfigurationTest.java

```diff
@@ -25,7 +22,7 @@ public void setUp()
         when(dummyEnvironmentManager.getEnvironmentVariable("SAUCE_ACCESS_KEY")).thenReturn("accessKey");
     }
 
-    public SauceSession instantiateSauceSession() {
+    SauceSession instantiateSauceSession() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/SauceLabs/simple_sauce/pullRequest?prid=4594401)